File: ticketProject7-CrossPlatform/TS-crossp/networking/response/AddBuyerResponse.cs
using System;
namespace networking.protocol
{

	using BuyerDTO = networking.dto.BuyerDTO;

	[Serializable]
	public class AddBuyerResponse : Response
	{
		private BuyerDTO buyer;

		public AddBuyerResponse(BuyerDTO buyer)
		{
			this.buyer = buyer;
		}

		public virtual BuyerDTO Buyer
		{
			get
			{
				return buyer;
			}
		}
	}

}

File: ticketProject4-DistributedJava/TicketSystem/TicketNetworking/src/main/java/network/objectprotocol/AddBuyerResponse.java
package network.objectprotocol;

import network.dto.BuyerDTO;

public class AddBuyerResponse implements Response {
    private BuyerDTO buyer;

    public AddBuyerResponse(BuyerDTO buyer) {
        this.buyer = buyer;
    }

    public BuyerDTO getBuyer() {
        return buyer;
    }
}
